tests/e2e/test_rosacli_ingress.go

```diff
@@ -403,4 +404,65 @@ var _ = Describe("Edit default ingress",
 					Expect(expectLabel).To(BeElementOf(ingressRouteSelectors))
 				}
 			})
+		It("can update ingress components (oauth, downloads, console) - [id:72868]",
+			labels.Medium,
+			labels.Runtime.Day2,
+			func() {
+
+				By("Record ingress default value")
+				output, err := rosaClient.Ingress.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				ingressList, err := rosaClient.Ingress.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				defaultIngress := ingressList.Ingresses[0]
+
+				By("Check edit ingress help message")
+				output, err = rosaClient.Ingress.EditIngress(clusterID, "-h")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("--component-routes"))
+
+				By("Edit ingress with --component-routes")
+				componentRoutes := "oauth: hostname=oauth.hostname.com;tlsSecretRef=oauth-secret,downloads: hostname=downloads.hostname.com;tlsSecretRef=downloads-secret,console: hostname=console.hostname.com;tlsSecretRef=console-secret"
+				output, err = rosaClient.Ingress.EditIngress(clusterID,
+					defaultIngress.ID,
+					"--component-routes", componentRoutes,
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = rosaClient.Ingress.EditIngress(clusterID,
+					defaultIngress.ID,
+					"--component-routes", componentRoutes,
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("List ingress to check")
+				output, err = rosaClient.Ingress.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Edit ingress with --component-routes with incorrect syntax")
+				componentRoutes = "oauth: hostname:custom1;tlsSecretRef=custom1,downloads: hostname=custom2;tlsSecretRef=custom2,console: hostname=custom3;tlsSecretRef=custom3"
+				output, err = rosaClient.Ingress.EditIngress(clusterID,
+					defaultIngress.ID,
+					"--component-routes", componentRoutes,
+				)
+				Expect(err).To(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("An error occurred whilst parsing the supplied component routes: only the name of the component should be followed by ':'"))
+
+				By("List ingress to check")
```

Comment:
If last steps is expecting an error happen, no need to list the ingress for double checking